test/grouped_convnd_fwd/test_grouped_convnd_fwd.cpp

```diff
@@ -41,7 +41,7 @@ class TestGroupedConvndFwd : public ::testing::Test
                                true,  // do_verification
                                1,     // init_method: integer value
                                false, // do_log
-                               false, // time_kernel
+                               true,  // time_kernel
```

Comment:
by defualt should be disabled in tests

Reply:
will disable timer
Another pull request will be created soon thank you!